src/js/tech/html5.js

```diff
@@ -127,7 +127,7 @@ class Html5 extends Tech {
 
       // If the original tag is still there, clone and remove it.
       if (el) {
-        const clone = el.cloneNode(false);
+        const clone = el.cloneNode(true);
```

Comment:
Otherwise, we won't get tracks available on something like ios.
Previously, we'd grab all the tracks from the element and store them on the `options` object (which still happens) but then we need to pass it to the tech. However, if we pass through both that option and the `textTracks` object, if you swap between techs, you'll start getting duplicated tracks. So, instead, I can remove the need to pass through the `tracks` option to the tech and just clone the node with its sub nodes, since the `tracks` option should only have been used the first time the player was initialized.